controllers/topics.controller.js

```diff
@@ -0,0 +1,11 @@
+const { selectTopics } = require("../models/topics.model.js");
+
+exports.getTopics = (request, response, next) =>
+{
+    selectTopics()
+        .then((topics) =>
+        {
+            response.status(200).send({ topics })
+        })
+        .catch(next);
```

Comment:
Really nice to see a .catch added here, just incase something goes wrong we can still deal with it.